torch/csrc/dynamo/eval_frame.c

```diff
@@ -612,7 +614,9 @@ static PyObject* _custom_eval_frame(
   if (callback == Py_False) {
     DEBUG_TRACE("In run only mode %s", get_frame_name(frame));
     _PytorchRecordFunctionState* rf = _pytorch_record_function_enter(cache_lookup_profiler_str);
-    PyObject* maybe_cached_code = lookup(extra, locals, backend);
+    PyObject* maybe_cached_code = NULL;
+    const char* trace_annotation = NULL;
```

Comment:
I don't know if _pytorch_record_function_enter works with NULL, but if it doesn't it'd be safer to put an empty string constant here.

Reply:
Decided to initialize them as "" and only change them in lookup if a valid cache entry is found.